programs/stake/src/lib.rs

```diff
@@ -2,15 +2,16 @@
 #![allow(clippy::integer_arithmetic)]
 use solana_sdk::genesis_config::GenesisConfig;
 
+#[deprecated(
+    since = "1.8.0",
```

Comment:
Will we be backporting this to v1.7 at all? Or stake pools can wait that long?

Reply:
I forgot to tag it, but yes I would like to backport to 1.7.  Does this need to change to accommodate that?

Reply:
Yeah, I'd put v1.7.2 here.

Reply:
great, thanks!